packages/airnode-utilities/src/caching/index.ts

```diff
@@ -202,6 +203,35 @@ const init = () => {
   sweep();
 };
 
+/**
+ * Calls the OS syncs command to forcefully sync the cache filesystem.
+ * This is the functional equivalent of "commit" from SQL.
+ */
+export const syncFsSync = () => {
+  try {
+    spawn(`sync`, [`-f`, CACHE_BASE_PATH]);
+  } catch (e) {
+    logger.error(`Unable to sync cache fs`);
+    logger.error((e as Error).message);
+  }
+};
+
+/**
+ * Syncs the temporary filesystem asynchronously.
+ * Lambda will "freeze" and "thaw" the execution context and with it any async promises in progress. We create and
+ * forget this promise here, but it will resolve at some point, either in this invocation or a later one.
+ */
+export const syncFsASync = () => {
```

Comment:
I am not sure I understand why would you want to call this asynchronously.

Reply:
I see your point and would agree it seems sync is what we need to ensure predictability, but @aquarat may have had a reason for using it `start-coordinator` in his original implementation? https://github.com/api3dao/airnode-qrng/blob/75c54a0f5dddbd2a29b7f303310356750d2fd1fc/packages/airnode-node/src/handlers/start-coordinator.ts#L120

Reply:
I'd say it was for performance. Basically, let Airnode continue processing the API responses as soon as possible and sync the FS sometime later. But I'd argue whether it helps. We may confirm this later when @aquarat comes back.

For now, I suggest using the sync version (and removing the async one completely). I don't want you blocked on this.

Reply:
> I suggest using the sync version (and removing the async one completely)

Done

Reply:
Exactly as @Siegrift assumed.
Lambda's `tmpfs` is extremely fast, so `sync` vs `async` won't have a noticeable performance impact and `sync` is safer anyway. :)

Reply:
Great, thanks @aquarat!